src/models/data_table_spec.ts

```diff
@@ -0,0 +1,98 @@
+import { describe, it } from 'mocha'
+import { expect } from 'chai'
+import DataTable from './data_table'
```

Comment:
Out of interest, do we intend to at some point move this to the `datatable` package in the monorepo, alongside the Java one?

Reply:
True that will probably be a good idea. Need to look more into it to see if javascript can use the same principles without built int types